### DISCUSSION TITLE
Streaming iPhone 15 Pro stereo video (spatial video)

### DISCUSSION THREAD

**Discussion by shandroa-vimn** [HaishinKit/HaishinKit.swift#1441] [Q&A]
Hi,

Is it possible to use this solution to stream spatial video(stereo video) introduced by Apple in iPhone 15 Pro?
[https://www.apple.com/newsroom/2023/12/apple-introduces-spatial-video-capture-on-iphone-15-pro/](url)

I found an article on how to record spatial video using AVFoundation but not sure if it's possible to use that approach for video streaming via RTMP: [https://www.finnvoorhees.com/words/reading-and-writing-spatial-video-with-avfoundation](url)

Please advice

**Comment by paulhdk** [comment]
+1.

I would also be interested in whether this is possible or what the roadblocks are.
